### PR TITLE
PS-1 Create index.html

### PR DESCRIPTION
**Pull request by Yash8141**
This code is related to the demo of the git file how to push pull and qubal qubal qubal jodi of jenish ❤ bandwala

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+<h1>Git Demo ψ(｀∇´)ψψ(｀∇´)ψ</h1>
+<p>This is the way we can use the github</p>
```
Comment (jenish-d-7span) [inline index.html:12]: Make proper Indentation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+</body>
+
```
Comment (jenish-d-7span) [inline index.html:15]: remove this space

**Review by jenish-d-7span — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Yash8141) [reply index.html:12]: Done

File: index.html (hunk 2 above)
Comment (Yash8141) [reply index.html:15]: Done

**Review by jenish-d-7span — APPROVED** [review]
You can merge in develop

**Pull request merged**